repository_service_tuf_worker/repository.py

```diff
@@ -277,6 +277,10 @@ def _persist(self, role: Metadata, role_name: str) -> str:
 
         bytes_data = role.to_bytes(JSONSerializer())
         self._storage_backend.put(bytes_data, filename)
+        if role_name == Root.type:
+            self.write_repository_settings(
+                f"{Root.type.upper()}_SIGNING", None
```

Comment:
It is documented in the `write_repository_setting`

>         Repository settings are stored in a dictionary like Dynaconf object
>         and each of them has its own key and value.
>         Additionally, repository settings are persisted in the Redis server
>         so that they can be reused by multiple RSTUF Worker instances.

Dynaconf accept all types supported as well by Redis